app/views/dashboards/show.html.erb

```diff
@@ -6,11 +6,11 @@
 
 <h1><%= "#{t(@dashboard.user_role)}  #{t('dashboard')}" %></h1>
 
-<% if policy(:defence_request).create? %>
+<%# if policy(:defence_request).create? %>
```

Comment:
Is this supposed to be commented? If not, should we not just remove the if statement?